atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -521,12 +524,25 @@ def get_signing_intent_by_keys(self, keys):
         if isinstance(keys, six.text_type):
             keys = keys.split()
         keys = set(keys)
+        deprecated_intent_matches = []
```

Comment:
rename it to something else, since it basically contains intents with correct keys, which was based on keys including deprecated

maybe something like

intents_for_deprecated_keys

Reply:
Done